nitro-protocol/.prettierrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  trailingComma: "none",
```

Comment:
We could change this rule to generate a smaller diff when "fixing" the prettier warnings. 

https://prettier.io/docs/en/options.html#trailing-commas

The actual rule is not that important -- having _some_ rule is what we are really interested in.